src/diagram/components/ui/variable-chip.scss

```diff
@@ -1,10 +1,23 @@
-@use "../../../scss/vars.scss";
+@import "../../../scss/vars.scss";
+@import "../../../scss/mixins.scss";
```

Comment:
FYI: `@import` has been deprecated in favor of `@use`, so we should be migrating the other way moving forward. (I only learned about `@use` after seeing it in a previous PR of @tealefristoe's.)

Reply:
@kswenson Good to know. Will keep that in mind going forward 👍